CI is failing trying to run unit tests.

Run npm run test:unit -- --run
npm error Missing script: "test:unit"
npm error
npm error To see a list of scripts, run:
npm error   npm run
npm error A complete log of this run can be found in: /home/runner/.npm/_logs/2026-03-02T00_26_57_162Z-debug-0.log
Error: Process completed with exit code 1.

--- a/app/package.json
+++ b/app/package.json
@@ -11,7 +11,8 @@
 		"check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
 		"check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
 		"lint": "prettier --check . && eslint .",
-		"format": "prettier --write ."
+		"format": "prettier --write .",
+		"test:unit": "vitest"
 	},
 	"devDependencies": {
 		"@axe-core/playwright": "^4.11.1",
@@ -29,6 +30,7 @@
 		"@types/bcryptjs": "^2.4.6",
 		"@types/better-sqlite3": "^7.6.13",
 		"@vitest/browser": "^4.0.18",
+		"@vitest/browser-playwright": "^4.0.18",
 		"drizzle-kit": "^0.31.9",
 		"eslint": "^10.0.2",
 		"eslint-config-prettier": "^10.1.8",

--- a/app/vitest.config.ts
+++ b/app/vitest.config.ts
@@ -1,0 +1,16 @@
+import { svelte } from '@sveltejs/vite-plugin-svelte';
+import { playwright } from '@vitest/browser-playwright';
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+	plugins: [svelte()],
+	test: {
+		passWithNoTests: true,
+		browser: {
+			enabled: true,
+			provider: playwright(),
+			headless: true,
+			instances: [{ browser: 'chromium' }]
+		}
+	}
+});

